### PR TITLE
correct reference for runOnce-Strategy

### PR DESCRIPTION
**Pull request by DenWin**
Preferably examples for canary and rolling should be added, though I do not have them at this point in time

### CODE REVIEW

**Comment by PRMerger8** [comment]
@DenWin : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by juliakm** [comment]
Thanks @DenWin!
#sign-off

**Review by juliakm — APPROVED** [review]

**Comment by juliakm** [comment]
#sign-off

File: docs/pipelines/process/deployment-jobs.md [hunk 1]
```diff
@@ -378,8 +378,8 @@ To share variables between stages, output an [artifact](../artifacts/pipeline-ar
 
 While executing deployment strategies, you can access output variables across jobs using the following syntax.
 
-- For **runOnce** strategy: `$[dependencies.<job-name>.outputs['<job-name>.<step-name>.<variable-name>']]` 
-    - For deployment jobs using the **runOnce** strategy: `$[dependencies.<job-name>.outputs['<lifecycle-hookname>_<resource-name>.<step-name>.<variable-name>']]`. For example, if you have a deployment job to a virtual machine named `Vm1`, the output variable would be `$[dependencies.<job-name>.outputs['Deploy_vm1.<step-name>.<variable-name>']]`
+- For **runOnce** strategy: `$[dependencies.<job-name>.outputs['<lifecycle-hookname>.<step-name>.<variable-name>']]` (e.g. `$[dependencies.JobA.outputs['Deploy.StepA.VariableA']]` )
+- For **runOnce** strategy plus a resourceType: `$[dependencies.<job-name>.outputs['<lifecycle-hookname>_<resource-name>.<step-name>.<variable-name>']]`. (e.g. `$[dependencies.JobA.outputs['Deploy_VM1.StepA.VariableA']]` )
```
Comment (ShannonLeavitt) [inline docs/pipelines/process/deployment-jobs.md:382]:
```suggestion
- For **runOnce** strategy: `$[dependencies.<job-name>.outputs['<lifecycle-hookname>.<step-name>.<variable-name>']]` (for example, `$[dependencies.JobA.outputs['Deploy.StepA.VariableA']]`)
- For **runOnce** strategy plus a resourceType: `$[dependencies.<job-name>.outputs['<lifecycle-hookname>_<resource-name>.<step-name>.<variable-name>']]`. (for example, `$[dependencies.JobA.outputs['Deploy_VM1.StepA.VariableA']]`)
```
Edits for Azure style.

**Pull request merged**